### ISSUE TITLE
Change Request: Adding option to sync segment overrides across all enviroments

### ISSUE THREAD

**Issue by turgaygulmez** [SolidStateGroup/bullet-train-api#1472]
When adding segment override to particular feature on any enviroment, it would be great to have an option to apply the same segment override across all enviroments in order to avoid manual addition on other enviroments.

**Comment by matthewelwell** [comment]
It's possible (and perhaps preferable?) that this could be achieve on the front end only.

**Comment by kyle-ssg** [comment]
I don't think this should be done on the frontend. Also I think sync endpoints with segment overrides / flags could open up opportunities for CICD integrations / maybe tools like Jira.

**Comment by kyle-ssg** [comment]
@matthewelwell I'm wondering if we can utilise the versioning work for this.

For each version (even historical) I believe we could quite easily do the following?

- Promote to live
- Save to environment

**Comment by chris-richmond** [comment]
I'm preparing to write a script (or app potentially) in the interim that will copy over values for all features with the "promote" tag from ENV A -> ENV B. This CR would still be helpful because I'd like to have a way to do this within the Flagsmith dashboard. It'd also be nice to see a diff before applying the promotion operation. Bonus points if the user can unselect certain items as they review the diff.
